### PR TITLE
11.0 auth ldap groups

### PR DESCRIPTION
**Pull request by dsolanki-initos**

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]
Code review

File: users_ldap_groups/__manifest__.py [hunk 1]
```diff
@@ -1,11 +1,12 @@
-# -*- coding: utf-8 -*-
+# Copyright 2020 initOS GmbH
 # Copyright 2012-2018 Therp BV <https://therp.nl>
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 {
     "name": "LDAP groups assignment",
-    "version": "10.0.0.0.0",
+    "version": "11.0.0.0.0",
```
Comment (fshah-initos) [inline users_ldap_groups/__manifest__.py]: Version must be according to guideline

**Review by fshah-initos — APPROVED** [review]

**Comment by dsolanki-initos** [comment]
Is it possible to merge this PR?

**Comment by dsolanki-initos** [comment]
@pedrobaeza Will you please merge this PR?

**Pull request merged**